src/test/java/DirectionsClientTest.java

```diff
@@ -33,50 +29,48 @@
  */
 @RunWith(JUnit4.class)
-public class DirectionsClientImplTest {
-
-  private static final List<String> A_TO_B_NO_WAYPOINTS =
-      ImmutableList.of(
-          "[DirectionsLeg: \"A\" -> \"B\" (45,-73 -> 43,-80), duration=6 hours, distance=6 km: 2 steps]");
+public class DirectionsClientTest {
 
   @Test
-  public void aToBNoWaypoints() throws Exception {
-
-    // Create fake DirectionsResult to parse
-    Distance distance = new Distance();
-    distance.humanReadable = "6 km";
+  public void parseDirectionsResult() throws Exception {
     Duration duration = new Duration();
-    duration.humanReadable = "6 hours";
-
     DirectionsLeg leg = new DirectionsLeg();
-    leg.steps = new DirectionsStep[] {new DirectionsStep(), new DirectionsStep()};
-    leg.distance = distance;
+    DirectionsRoute route = new DirectionsRoute();
+    DirectionsRoute[] routes = {route};
+    DirectionsResult directionsResult = new DirectionsResult();
+
+    duration.humanReadable = "6 hours";
     leg.duration = duration;
-    leg.startLocation = new LatLng(45, -73);
-    leg.endLocation = new LatLng(43, -80);
-    leg.startAddress = "A";
     leg.endAddress = "B";
-
-    DirectionsRoute route = new DirectionsRoute();
     route.legs = new DirectionsLeg[] {leg};
+    directionsResult.routes = routes;
+
+    List<String> actual = DirectionsClient.parseDirectionsResult(directionsResult);
+
+    Assert.assertEquals(ImmutableList.of("6 hours to travel to B"), actual);
+  }
 
-    GeocodedWaypoint[] geocodedWaypoints = {new GeocodedWaypoint(), new GeocodedWaypoint()};
+  @Test
+  public void getTotalTravelTime() throws Exception {
+    Duration durationOne = new Duration();
+    Duration durationTwo = new Duration();
+    DirectionsLeg legOne = new DirectionsLeg();
+    DirectionsLeg legTwo = new DirectionsLeg();
+    DirectionsRoute route = new DirectionsRoute();
     DirectionsRoute[] routes = {route};
     DirectionsResult directionsResult = new DirectionsResult();
-    directionsResult.geocodedWaypoints = geocodedWaypoints;
-    directionsResult.routes = routes;
 
-    List<String> actual = DirectionsClientImpl.parseDirectionsResult(directionsResult);
+    durationOne.humanReadable = "6 seconds";
+    durationOne.inSeconds = 6;
+    durationTwo.humanReadable = "12 seconds";
+    durationTwo.inSeconds = 12;
+    legOne.duration = durationOne;
+    legTwo.duration = durationTwo;
+    route.legs = new DirectionsLeg[] {legOne, legTwo};
+    directionsResult.routes = routes;
 
-    // Trailing zeroes in tests omitted by default to reduce clutter
-    List<String> expected =
-        ImmutableList.of(
-            A_TO_B_NO_WAYPOINTS
-                .get(0)
-                .replace(
-                    "(45,-73 -> 43,-80)",
-                    "(45.00000000,-73.00000000 -> 43.00000000,-80.00000000)"));
+    long actual = DirectionsClient.getTotalTravelTime(directionsResult);
 
-    Assert.assertEquals(expected, actual);
+    Assert.assertEquals(18, actual);
```

Comment:
I would use `durationOne.inSeconds + durationTwo.inSeconds` here for clarity

Reply:
I agree, 18 is confusing. Thanks!

Reply:
`RESOLVED`